commit c323fced9a6890eeb6a885f4d9ff3f5e1d3cb3f5 (from e578f988d3a0c167947ba33a7207cc724c9a8a4e)
Merge: e578f98 d4d1540
Author: ErikZalm <erik@vdzalm.eu>
Date:   Wed Jul 24 02:49:08 2013 -0700

    Merge pull request #554 from iXce/cxa_atexit
    
    Fix Makefile-based build when avr-gcc has been compiled with --enable-cxa_atexit

diff --git a/Marlin/Makefile b/Marlin/Makefile
index a4ae1d7..a56bc1e 100644
--- a/Marlin/Makefile
+++ b/Marlin/Makefile
@@ -283,6 +283,7 @@ ifneq ($(HARDWARE_MOTHERBOARD),)
 CTUNING += -DMOTHERBOARD=${HARDWARE_MOTHERBOARD}
 endif
 #CEXTRA = -Wa,-adhlns=$(<:.c=.lst)
+CEXTRA = -fno-use-cxa-atexit
 
 CFLAGS := $(CDEBUG) $(CDEFS) $(CINCS) -O$(OPT) $(CWARN) $(CEXTRA) $(CTUNING)
 CXXFLAGS :=         $(CDEFS) $(CINCS) -O$(OPT) -Wall    $(CEXTRA) $(CTUNING)

